internal/addon/authentication/certificates.go

```diff
@@ -47,15 +47,13 @@ func CreateOrUpdateRootCertificate(k8s client.Client) error {
 }
 
 func checkCertManagerCRDs(ctx context.Context, k8s client.Client) error {
-	crds := []string{"certificates.cert-manager.io", "issuers.cert-manager.io", "clusterissuers.cert-manager.io"}
-
-	for _, crdName := range crds {
+	for _, crdName := range certManagerCRDs {
 		key := client.ObjectKey{Name: crdName}
 		crd := &apiextensions.CustomResourceDefinition{}
 		err := k8s.Get(ctx, key, crd, &client.GetOptions{})
 		if err != nil {
 			if errors.IsNotFound(err) {
-				return kverrors.New("cert-manager CRD is missing %s", crdName)
+				return kverrors.New(fmt.Sprintf("cert-manager CRD is missing %s", crdName))
```

Comment:
nit. kverrors is meant to be used as
```suggestion
				return kverrors.New("cert-manager CRD is missing", "name", crdName)
```